tests/core/test_scheming.py

```diff
@@ -239,7 +239,7 @@ def test_resolution():
 
 
 if __name__ == '__main__':
-    test_saider()
-    test_json_schema()
-    test_json_schema_dict()
+    # test_saider()
```

Comment:
Did you mean to leave these commented out?

Reply:
The __main__ in the tests is just for running in the IDE, pytest picks up all the tests and runs them regardless.  So I haven't paid attention to the __main__ methods after I'm done debugging the tests.

Reply:
I removed the comments